extensions/wikia/ArticleVideo/styles/video-attribution.scss

```diff
@@ -0,0 +1,44 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/colors';
+
+.featured-video-wrapper {
+	.attribution-container {
```

Comment:
how about video-attribution-container
and then
video-attribution-avatar
and so on
then we would not have to nest these that much